packages/suite/src/components/suite/QuestionTooltip/index.tsx

```diff
@@ -16,42 +16,27 @@ const Label = styled.div`
 interface Props {
     label?: JSX.Element | ExtendedMessageDescriptor['id'];
     tooltip?: JSX.Element | ExtendedMessageDescriptor['id'];
-    size?: number;
     className?: string;
-    color?: string;
-    // This is workaround for pixel-perfect alignment of the icon: see https://medium.com/microsoft-design/leading-trim-the-future-of-digital-typesetting-d082d84b202
-    iconStyle?: React.SVGAttributes<HTMLDivElement>['style'];
 }
 
-export const QuestionTooltip = ({
-    label,
-    tooltip,
-    size = 16,
-    color,
-    iconStyle,
-    className,
-}: Props) => {
-    const theme = useTheme();
-    const defaultColor = color ?? theme.TYPE_LIGHT_GREY;
+export const QuestionTooltip = ({ label, tooltip, className }: Props) => {
     return (
         <Wrapper className={className}>
-            {label && (
-                <Label>{typeof label === 'string' ? <Translation id={label} /> : label}</Label>
-            )}
-            {tooltip && (
-                <Tooltip
-                    placement="top"
-                    content={typeof tooltip === 'string' ? <Translation id={tooltip} /> : tooltip}
-                >
-                    <Icon
-                        useCursorPointer
-                        size={size}
-                        color={defaultColor}
-                        style={iconStyle}
-                        icon="QUESTION"
-                    />
-                </Tooltip>
-            )}
+            {label &&
```

Comment:
I think this changes the behaviour, before you were able to pass a tooltip (content) without providing a label. If I understand correctly that is not possible in this new version. Is it intentional?

Oh I see, that's probably side-effect of new styles where we underline the text that triggers the tooltip!

Reply:
I think we can remove this component, its purpose was to streamline showing tooltip on mouse hover over question-mark icon, since we are deleting it I'm not sure it if still does something useful. Could you please double check that and remove it if possible?

Reply:
Yes, if the new tooltips are accepted by @matejzak this component is probably obsolete. I'd maybe do refactoring of this particular component in another PR.